lib/doorkeeper-mongodb/mixins/mongoid/base_mixin.rb

```diff
@@ -9,6 +9,14 @@ def ordered_by(attribute, direction = :asc)
             order_by(attribute => direction.to_sym)
           end
         end
+
+        def as_json(*args)
+          json_response = super
+
+          json_response["id"] = json_response["_id"]
```

Comment:
Why do we need this information? :thinking: Original Doorkeepr token mixin doesn't return internal ID column in JSON response

Reply:
There was a spec in doorkeeper that was specifically testing for the field `id`.  When Mongoid serializes it only serialized a `_id`.  I added this to allow doorkeeper specs to pass.

Other things changed since I wrote this PR, so maybe this has changed too.  If I can get tests to pass without it, I'll take it out.

You asked though, so that is why.